Source/NSString.m

```diff
@@ -6248,6 +6256,156 @@ - (BOOL) containsString: (NSString *)string
   return [self rangeOfString: string].location != NSNotFound;
 }
 
+- (void) enumerateSubstringsInRange: (NSRange)range 
+                            options: (NSStringEnumerationOptions)opts 
+                         usingBlock: (GSNSStringEnumerationBlock)block
+{
+  // Get low byte.
+  uint8_t substringType = opts & 0xFF;
+
+  BOOL isReverse = opts & NSStringEnumerationReverse;
+  BOOL substringNotRequired = opts & NSStringEnumerationSubstringNotRequired;
+  BOOL localized = opts & NSStringEnumerationLocalized;
+  
+  NSUInteger currentLocation;
+  BOOL stop = NO;
+  if(isReverse) {
```

Comment:
I would prefer a bit more whitespace here. That is an empty line before and a space between `if` and the bracket. The later is true for a lot more places below.

Reply:
Are you saying I should switch to the GNU coding standards, or should I just add whitespace to my existing format?

Reply:
Contributions should ideally match the existing coding style (otherwise we'll just need to reformat after the merge).